tests/security_features_test.go

```diff
@@ -141,6 +142,30 @@ var _ = Describe("SecurityFeatures", func() {
 				}
 				Expect(*container.SecurityContext.Privileged).To(BeFalse())
 			})
+
+			It("[test_id:5297]Make sure qemu processes are MCS constrained", func() {
+
+				By("Starting a VirtualMachineInstance")
+				vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred())
+				tests.WaitForSuccessfulVMIStart(vmi)
+
+				pod := tests.GetRunningPodByVirtualMachineInstance(vmi, tests.NamespaceTestDefault)
+
+				qemuProcess, err := tests.ExecuteCommandOnPod(
```

Comment:
Nit? The variable `qemuProcess` contains the SELinux context of the qemu process, its name should probably reflect that.

Reply:
Agreed, updated to `qemuProcessSelinuxContext`